cmd/thv/app/run.go

```diff
@@ -213,7 +211,7 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 	cmdArgs := parseCommandArguments(os.Args)
 
 	// Print the processed command arguments for debugging
-	logger.Infof("Processed cmdArgs: %v", cmdArgs)
+	logger.Debugf("Processed cmdArgs: %v", cmdArgs)
```

Comment:
Another incidental change: use debug logging for this since it is noisy otherwise.